.eleventy.js

```diff
@@ -357,24 +359,36 @@ module.exports = function (eleventyConfig) {
           ${demoDates
             .map(date => {
               if (validDates.includes(date.date)) {
-                const options = {
+                const dateOptions = {
                   weekday: 'long',
                   year: 'numeric',
                   month: 'long',
                   day: 'numeric',
                   timeZone: 'UTC',
                 };
-                const prnDt = new Date(date.date).toLocaleString(
-                  locale,
-                  options,
+
+                const timeOptionsEN = {
+                  hour: 'numeric',
+                  minute: '2-digit',
+                  hour12: true,
+                };
+
+                const timeOptionsFR = {
```

Comment:
Optional comment: Instead of having separate `timeOptionsEN` and `timeOptionsFR` objects, you could create a function that dynamically generates the options based on the locale. This way, it can be expanded to support other languages or locales more easily if we decide to do so in the future.